Python/gc_free_threading.c

```diff
@@ -308,6 +309,27 @@ gc_visit_heaps(PyInterpreterState *interp, mi_block_visit_fun *visitor,
     return err;
 }
 
+static void
+gc_visit_thread_stacks(struct _stoptheworld_state *stw)
+{
+    HEAD_LOCK(&_PyRuntime);
+    PyInterpreterState *interp;
+    PyThreadState *tstate;
+    _Py_FOR_EACH_THREAD(stw, interp, tstate) {
+        _PyStackChunk *curr_chunk = tstate->datastack_chunk;
+        while (curr_chunk != NULL) {
+            for (size_t curr_i = 0; curr_i < curr_chunk->top; curr_i++) {
+                _PyStackRef curr_o = Py_STACK_TAG_UNSAFE(curr_chunk->data[curr_i]);
+                if ((curr_o.bits & Py_TAG_DEFERRED) == Py_TAG_DEFERRED) {
+                    gc_add_refs(Py_STACK_UNTAG_OWNED(curr_o), 1);
```

Comment:
This should be something like `Py_STACK_UNTAG_BORROWED`. We don't want to actually increase the refcount of the object -- just the computed gc_refs (i.e., the refcount - incoming_references).

Reply:
Yeah woops it was a bunch of testing code I committed. Thanks for pointing that out!